constructr-coordination-redis/src/test/scala/com/lightbend/constructr/coordination/zookeeper/ZookeeperCoordinationSpec.scala

```diff
@@ -1,77 +0,0 @@
-package com.lightbend.constructr.coordination.zookeeper
```

Comment:
This file is part of the `zookeeper-coordination-redis` project, which was a left over from copy & paste ;-)